org-code-javabuilder/protocol/src/main/java/org/code/protocol/ContentManager.java

```diff
@@ -0,0 +1,10 @@
+package org.code.protocol;
+
+public interface ContentManager {
```

Comment:
Nice! Can we add a quick javadoc here? something like "Manages content (such as code sources and assets) that are used and generated during a Javabuilder session"